commerce-openapi-core-impl/build.gradle

```diff
@@ -29,5 +30,4 @@ deployDependencies {
 
 	include "aalto-xml-*.jar"
 	include "jackson-*.jar"
```

Comment:
@marco-leo if we can leave `jackson-*` dependencies as is it will work until some other modules start using it this way.